models/base/snowplow_base_events.sql

```diff
@@ -1,2 +1,47 @@
+with source as (
 
-select * from {{ var('snowplow:events') }}
+    select * from
+
+    {% if var('snowplow:use_fivetran_interface') %}
```

Comment:
I did, I actually tested both branches. I'm not super-familiar with the limitation you're talking about but I know it works in both conditionals here.